Enable `enable_new_opmath` for `lightning.tensor` (#919)
PennyLaneAI/pennylane-lightning: .github/CHANGELOG.md

### Bug fixes

> * Bug fix for `_pauli_word` of `QuantumScriptSerializer`. `_pauli_word` can process `PauliWord` object: `I`.

Reviewer: ```suggestion
* Bug fix in `_pauli_word` of `QuantumScriptSerializer`. `_pauli_word` can now process empty `PauliWord` observables. 
```

Reviewer: Ohh, gotta!
